### DISCUSSION TITLE
How can I push my code?

### DISCUSSION THREAD

**Discussion by cxy-1993** [community/community#119343] [Code Security]
### Select Topic Area

Question

### Body

When I tried to push the code to the repository, I encountered an issue. Here is the process of what happened:

I have forked a repo from llvm-project, made some modifications ("main branch in https://github.com/cxy-1993/llvm-project"). After rebase master, I can't push codes to main branch. Here is the error log:

```
 ! [remote rejected]           fuse-consumer -> main (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/email-check.yaml` without `workflow` scope)
error: failed to push some refs to 'https://github.com/cxy-1993/llvm-project.git'
```

I tried to follow instructions(https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule) to make the branch unprotected, but I can't find branch button in "Code and automation" section of the sidebar. 

I also find related discussion here(https://github.com/orgs/community/discussions/26254). Neither generate new tokens nor use github cli were of any help to me.

So how can I push my code, just push force is enough for me. Any help is appreciated.

**Comment by KittyChiu** [comment]
The error is saying the PAT you are using to push the code does not have sufficient permission. Because you are modified (I guess) a workflow YML file, you need to review your PAT to include `workflow` permission.

**Reply by ofinto** [reply]
I have the same problem, but I work with Git using SSH. As I understand, I should generate a PAT and switch to HTTPS

**Comment by cxy-1993** [comment]
Thank you for your reply, I was able to resolve the issue using SSH.

**Reply by ofinto** [reply]
Could you post how you solved this?
